### PR TITLE
Service files metadata

### PR DESCRIPTION
**Pull request by Zapadlo**

### CODE REVIEW

**Comment by Nesze** [comment]
+1

File: ansible/userdata/default_instance_user_data.yaml [hunk 1]
```diff
@@ -58,4 +63,33 @@ write_files:
...
+      After=docker.service
+      Requires=docker.service
+
+      [Service]
+      Environment="DOCKER_APP_VERSION=service-files-metadata"
+      TimeoutStartSec=0
+      Type=oneshot
+      ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p > /dev/null 2>&1'
+      ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p > /dev/null 2>&1'
+      ExecStartPre=/bin/bash -c '/usr/bin/docker pull angelx/coco-fleet-deployer:$DOCKER_APP_VERSION'
```
Comment (Nesze) [inline ansible/userdata/default_instance_user_data.yaml]: angelx ?

**Comment by martingartonft** [comment]
lgtm apart from angelx that Nesze already mentioned

File: ansible/userdata/default_instance_user_data.yaml (hunk 1 above)
Comment (Zapadlo) [inline ansible/userdata/default_instance_user_data.yaml]: Yep, fixed, back at 'coco'

**Pull request merged**